### ISSUE TITLE
XCFramework is missing in the release

### ISSUE THREAD

**Issue by FranzBusch** [BlinkID/blinkid-swift-package#1]
@juraskrlec The XCFramework is not attached to the release. When I tried to add the package to our project it failed.

**Comment by juraskrlec** [comment]
Hi @FranzBusch 

Have you followed our guide [here](https://github.com/BlinkID/blinkid-ios#using-swift-package-manager)? Our xcframework is attached to the release [v5.9.0 release](https://github.com/BlinkID/blinkid-ios/releases/tag/v5.9.0). Can you please copy paste me the error?

**Comment by FranzBusch** [comment]
@juraskrlec Yes I followed your guide. The problem is actually a different one. Yes the binary is attached to the release but the zip contains a wrong format. When unzipping there are multiple folders and inside a deep nesting there is the `.xcframework` according to the SwiftPM format the `.xcframework` should be at the root of the zip.

**Comment by juraskrlec** [comment]
Thank you for the details, I'm going to check that out as soon as possible.

**Comment by mijo-gracanin** [comment]
Hi @FranzBusch ,
can you please check again if issue is fixed by https://github.com/BlinkID/blinkid-swift-package/pull/2 ?

**Comment by mikeberlin** [comment]
We are having the exact same issue with the latest release 5.11.0. Will this be resolved soon? We cannot upgrade past 5.9.0 which is a blocker for our team.

**Comment by mijo-gracanin** [comment]
Hi @mikeberlin, as I mentioned here: https://github.com/BlinkID/blinkid-ios/issues/341#issuecomment-819347096, the issue should be resolved now.
Suppose you get some other error like TSCBasic.StringError, please delete the derived data.

**Comment by mikeberlin** [comment]
Perfect, thank you!
